clients/mobile/src/setup/navigation/BottomNavigation.tsx

```diff
@@ -25,11 +36,11 @@ const BottomNavigation = () => {
           <Ionicons name="analytics-outline" color={color} size={size} />
         )
       }} />
-      <Tab.Screen name={SCREENS.PROFILE} component={Profile} options={{
-        headerTitle: 'Profile',
+      <Tab.Screen name='Account' component={Account} options={{
```

Comment:
wasn't SCREENS.PROFILE` better approach?

Reply:
That was a mistake. Thanks for finding out. Changed accordingly